frontend/src/components/cluster-dashboard/Cluster.model.tsx

```diff
@@ -0,0 +1,22 @@
+export interface ClusterMember {
+  node: string;
+  nodeUid: string;
+  roles: string[];
+  status: string;
+}
+
+export class Cluster {
```

Comment:
Don't tell Charles you're making classes lol.

Reply:
Lmao, probably too late.

Reply:
😲